rake/dkdeploy-typo3-cms/typoscript.md

```diff
@@ -90,7 +90,7 @@ set :typoscript_config_file, 'filename.txt'
  {% highlight shell-session %}
  cap <stage> typo3:cms:typoscript:merge_configs[:typoscript_config_paths,:typoscript_config_file]
  {% endhighlight %}
- This task is called by ```merge_config_in_base_path```, so you may not need to call it yourself.
+ This task is called by <a href="#mergeconfiginbasepath">merge_config_in_base_path</a>, so you may not need to call it yourself.
```

Comment:
please remove all of the hrefs, we might about hyperlinks to other tasks for the whole page in a separate issue

just the task name with single back ticks :)